src/app/quiz/models/comment.ts

```diff
@@ -1,7 +1,9 @@
 export type CommentType = {
```

Comment:
```suggestion
export interface CommentProps {
```

는 어떠신가요? type은 단일 타입할 때 쓰면 어떨까요?

Reply:
옹 그러네요 이 부분 interface로 수정할게요!

Reply:
https://github.com/depromeet/toks-web/pull/304/commits/bbfd89f78cca3fdae5101e18428395f7e61f55ba
수정했습니다!